data/crac/crac-result-extensions/pom.xml

```diff
@@ -25,6 +25,11 @@
             <artifactId>farao-crac-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.farao-community.farao</groupId>
```

Comment:
For some reason, I can't compile if I put ${project.groupId} here, instead of com.farao-community.farao. Why?